pkg/lib/dynatrace.go

```diff
@@ -203,7 +203,10 @@ func (dt *DynatraceHelper) sendDynatraceAPIRequest(apiPath string, method string
 	defer resp.Body.Close()
 	responseBody, _ := ioutil.ReadAll(resp.Body)
 
+	dt.Logger.Debug("Dynatrace service returned status " + resp.Status)
```

Comment:
I've added some debug logging in the `sendDynatraceAPIRequest` method to ensure that we get error messages etc... for troubleshooting